File: 2.0/csharp/me/verifyCallerId.cs
using CallfireApiClient;
using CallfireApiClient.Api.Account.Model.Request;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var request = new CallerIdVerificationRequest
        {
            CallerId = "12132000384",
            VerificationCode = "1234"
        };
        bool? verified = client.MeApi.VerifyCallerId(request);
    }
}

File: 2.0/java/me/verifyCallerId.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.account.model.request.CallerIdVerificationRequest;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient callfireClient = new CallfireClient("api_login", "api_password");
        String phoneNumber = "12132000384";
        CallerIdVerificationRequest request = CallerIdVerificationRequest.create()
            .callerId(phoneNumber)
            .verificationCode("1234")
            .build();
        Boolean verified = callfireClient.meApi().verifyCallerId(request);
    }
}
